tests/queries/0_stateless/02751_ip_types_aggregate_functions_states.sql.j2

```diff
@@ -0,0 +1,172 @@
+{% set ip4_generator = "select number::UInt32::IPv4 ip from numbers(999999999,50) order by ip" %}
```

Comment:
Maybe it is worth to add top-level comment? Something like:
Test to validate that state representation of aggregation functions computed over IPv4 / IPv5 doesn't change. 
If you see this test failing, that might indicate breaking backward-compatibility of those functions with the previous CH version.
In that case, please consider upgrade/downgrade scenarios for the users.

Reply:
it fails because of https://github.com/ClickHouse/ClickHouse/issues/50549 https://github.com/ClickHouse/ClickHouse/issues/49410 https://github.com/ClickHouse/ClickHouse/pull/50551
this is PR is not for merge.